downstream/modules/platform/proc-aap-platform-gateway-backup.adoc

```diff
@@ -1,3 +1,33 @@
 [id="aap-platform-gateway-backup_{context}"]
 
-= Backing up your AnsibleAutomationPlatform resource
+= Backing up your AnsibleAutomationPlatform deployment
+Regularly backing up your *AnsibleAutomationPlatform* deployment is vital to protect against unexpected data loss and application errors. *AnsibleAutomationPlatform* hosts any enabled components (such as, {ControllerName}, {HubName}, {EDAName}, and {LightspeedShortName}), when you back up *AnsibleAutomationPlatform* you also back up these components.
+
+.Prerequisites
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been installed to the cluster.
+* The *AnsibleAutomationPlatform* instance is deployed using the {OperatorPlatform}.
+
+.Procedure 
+. Log in to {OCP}.
+. Go to menu:Operators[Installed Operators].
+. Select the {OperatorPlatform} installed on your project namespace.
+. Go to your btn:[All Instances] tab, and click btn:[Create New].
```

Comment:
I don't think tabs are classified as buttons. I think you just select them, or treat them as menu:...